modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -2507,7 +2511,13 @@
 			-- handle special "latest" version
 			if min == "latest" then
 				-- vs2015 and lower can't build against SDK 10
-				min = iif(_ACTION >= "vs2017", m.latestSDK10Version(), nil)
+				-- vs2019 allows for automatic assignment to latest
+				-- Windows 10 sdk if you set to "10.0"
```

Comment:
I haven't had a chance to try out VS2019 yet, is this a fully documented value? Can it be chosen in the dropdown in the GUI? If not, perhaps this should be `auto` and `latest` should stay as it was?

Reply:
Yes and it is set by default in projects. I think this move was due to the consistent addition of Windows SDK versions and how it would conflict with builds on different machines. 
Since its reasonable to expect the same (if not more) amount of sdk changes, they added this as a choice.

And yes this a choice you can make in the drop down menu 😁.